### ISSUE TITLE
Conflict with Recalculate normals with NDMF package lilNDMFMeshSimplifier

### ISSUE THREAD

**Issue by Spokeek** [hai-vr/prefabulous-avatar#8]
Hello, i was making use of both **Prefabulous** and the new upcoming package https://github.com/lilxyzw/lilNDMFMeshSimplifier from Lilxyzw

As i was doing so, i realized that there may be an issue with the **NDMF ordering queue**.

The Prefabulous component for **recalculate normals** seem to execute in the transforming phase. While this seems logical, optimization tools that transform the mesh are executed **in the step after** as seen in the **NDMF sequence**
![image](https://github.com/user-attachments/assets/48d3bc81-aeba-4fab-9774-415ebe573315)

I have a doubt if the solution is to move the step, or if that needs to be discussed within the NDMF system, but wanted to mention the issue.

Here are screenshots of a lasyusha when applying the recalculate component (on the right)
![image](https://github.com/user-attachments/assets/3994c5f0-78ca-4f9a-8f7e-fd3c80cfc281)

**Comment by hai-vr** [comment]
I've already moved Delete Polygons from the Optimizing phase to the Transforming phase in order to make it compatible with VRCFury's Blendshape Optimizer.

I cannot justify moving mesh transformation tools back and forth between the phases any more, the solution has to be something else than that.

**Comment by Spokeek** [comment]
Absolutely. I imagine we can upstream the issue on the NDMF side. I wasn't aware of that older change. Thank you for adding details. I'll go make an issue entry in that sense.

**Comment by Spokeek** [comment]
Upstreamed in a discussion at the NDMF level https://github.com/bdunderscore/ndmf/discussions/447
